src/features/content-explorer/content-explorer/ContentExplorerFolderTreeBreadcrumbs.js

```diff
@@ -0,0 +1,64 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { injectIntl } from 'react-intl';
+
+import Button from '../../../components/button';
+import DropdownMenu from '../../../components/dropdown-menu/DropdownMenu';
+import IconBreadcrumbArrow from '../../../icons/general/IconBreadcrumbArrow';
+import IconFolderTree from '../../../icons/general/IconFolderTree';
+import { Menu, MenuItem } from '../../../components/menu';
+
+import { FoldersPathPropType } from '../prop-types';
+import messages from '../messages';
+import { bdlGray50 } from '../../../styles/variables';
+
+const ContentExplorerFolderTreeBreadcrumbs = ({
+    foldersPath,
+    intl: { formatMessage },
+    isFolderTreeButtonDisabled = false,
+    numTotalItems,
+    onBreadcrumbClick,
+}) => (
+    <div className="content-explorer-folder-tree-container">
+        <DropdownMenu>
+            <Button
+                aria-label={formatMessage(messages.clickToGoBack)}
+                className="content-explorer-folder-tree-button"
+                type="button"
+                title="file path"
+                isDisabled={isFolderTreeButtonDisabled}
+            >
+                <IconFolderTree height={18} width={18} />
+            </Button>
+
+            <Menu>
+                {foldersPath.map((folder, i) => (
+                    <MenuItem
+                        data-testid="breadcrumb-lnk"
```

Comment:
Is this supposed to be `breadcrumb-link`?

Reply:
Yeah that's a good point. I will actually rename it to folder-tree-breadcrumb-link because there is another data-testid called this in the ContentExplorerBreadcrumbs file